app/helpers/application_helper.rb

```diff
@@ -4,7 +4,7 @@
 module ApplicationHelper
 
   def format_time(time = Time.now)
-    I18n.l(time, :format => "%d.%m.%Y %H:%M") unless time.nil?
+    I18n.l(time, format: :short) unless time.nil?
```

Comment:
This leaves out the year, I think we need to include it in the default time format.

Reply:
a fair point.  let me look closer to see what a better default is.  alternatively, these could (perhaps should) be specified in `locales/en.yml` etc

Reply:
Yes, I'd be ok adding the year to the locale files for the short format.

Reply:
I think it can lead to confusion if the `:short` format will be extended via locales. Maybe another approach is to introduce a new date format. Please have a look at #997.